main.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+    required_providers {
+      aws = {
+        source = "hashicorp/aws"
+        version = "~> 4.16"
+      }
+    }
+    required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+    region = "us-east-1"
+}
+
+resource "aws_instance" "app_server" {
+    ami           = "ami-0715c1897453cabd1"  # Amazon Linux 2023 AMI, free tier eligible official Amazon linux image.
+    instance_type = "t3.micro" 
+}
```

Comment:
could you  please, add newline at the end of file.

Reply:
Added a new line EOF.

Reply:
@geoffrey1330 
This `main.tf` file will be deleted. 
`.gitignore` is updated with the new line. 
Can you please approve this PR?